### PR TITLE
Misc fixes for APV

### PR DESCRIPTION
**Pull request by JulienIgnace-Unity**
### Purpose of this PR
Collection of small fixes for APV
- With a null ref exception when applying virtual offset on unloaded scenes.
- Fixed an issue causing failure of the bake when multiple scenes are loaded at the same time.
- Fixed debug probes after domain relaod when using L1
- Fixed per probe culling.

### CODE REVIEW

File: com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.VirtualOffset.cs [hunk 1]
```diff
@@ -17,6 +17,9 @@ private static void AddOccluders()
             for (int sceneIndex = 0; sceneIndex < UnityEngine.SceneManagement.SceneManager.sceneCount; ++sceneIndex)
             {
                 UnityEngine.SceneManagement.Scene scene = UnityEngine.SceneManagement.SceneManager.GetSceneAt(sceneIndex);
+                if (!scene.isLoaded)
```
Comment (FrancescoC-unity) [inline com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.VirtualOffset.cs:20]: Oh, weird that a scene is in the scene manager but is not loaded, did not know that was possible :D

File: com.unity.render-pipelines.high-definition/Runtime/Debug/ProbeVolumeDebug.shader [hunk 2]
```diff
@@ -65,7 +65,7 @@ Shader "Hidden/HDRP/ProbeVolumeDebug"
             float4 position = float4(UNITY_MATRIX_M._m03_m13_m23, 1);
             int brickSize = UNITY_ACCESS_INSTANCED_PROP(Props, _IndexInAtlas).w;
 
-            if (distance(position.xyz, _WorldSpaceCameraPos.xyz) > _CullDistance ||
+            if (distance(position.xyz, GetCurrentViewPosition()) > _CullDistance ||
```
Comment (FrancescoC-unity) [inline com.unity.render-pipelines.high-definition/Runtime/Debug/ProbeVolumeDebug.shader:68]: Uhm I am not sure I understand this. The position above should be absolute world position and as such we could compare with absolute position whereas this change will compare against 0 when camera relative is on.

File: com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.VirtualOffset.cs (hunk 1 above)
Comment (JulienIgnace-Unity) [reply com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.VirtualOffset.cs:20]: Had that by just clicking on a scene in the hierarchy and click "Unload".

File: com.unity.render-pipelines.high-definition/Runtime/Debug/ProbeVolumeDebug.shader (hunk 2 above)
Comment (JulienIgnace-Unity) [reply com.unity.render-pipelines.high-definition/Runtime/Debug/ProbeVolumeDebug.shader:68]: We can talk about this one when you're back. Was actually quite easy to repro.

**Review by FrancescoC-unity — APPROVED** [review]

**Pull request merged**